docs/plugins.md

```diff
@@ -0,0 +1,352 @@
+
+### What is a Lighthouse Plugin?
+
+Lighthouse plugins are a way to extend the functionality of Lighthouse with insight from domain experts (that's you!) and easily share this extra functionality with other Lighthouse users. At its core, a plugin is a node module that implements a set of checks that will get run by Lighthouse and added to the report as a new category.
+
+### Do You Need a Custom Audit?
+
+Lighthouse has a lot of audits already. It's possible the data is already being returned by Lighthouse! If you're in doubt about whether your use case needs a custom audit, [file an issue to discuss](https://github.com/GoogleChrome/lighthouse/issues/new/choose).
+
+Consider the criteria
+
+### Plugins v. Custom Config
+
+Plugins are easily shared and have a stable API that won't change between minor version bumps but are also more limited in scope. You might need a [custom Lighthouse configuration](./configuration.md) instead. Before getting started with plugins, think about your current needs, and consult the table below to decide which is best for you.
+
+| Capability                           | Plugins | Custom Config |
+| ------------------------------------ | ------- | ------------- |
+| Add custom audits                    | ✅      | ✅            |
+| Add custom categories                | ✅      | ✅            |
+| Easily sharable via NPM              | ✅      | ❌            |
+| Semver-stable API                    | ✅      | ❌            |
+| Gather custom data from audited page | ❌      | ✅            |
+| Modify core categories               | ❌      | ✅            |
+| Modify `config.settings` properties  | ❌      | ✅            |
+
+### Getting Started
+
+To develop a Lighthouse plugin, you'll need to write three things:
+
+1. A `package.json` file to define your plugin's dependencies and point to.
+1. A `plugin.js` file to configure your plugin's audits, category name, and scoring.
+1. Custom audit files that will contain the primary logic of the checks you want to perform.
+
+To see a fully functioning example, see our [plugin recipe](./recipes/lighthouse-plugin-example/readme.md).
+
+#### `package.json`
+
+A Lighthouse plugin is just a node module that starts with `lighthouse-plugin-`. Any dependencies you need are up to you, but do not depend on Lighthouse directly, use [`peerDependencies`](http://npm.github.io/using-pkgs-docs/package-json/types/peerdependencies.html) instead.
+
+**Example `package.json`**
+
+```json
+{
+  "name": "lighthouse-plugin-cats",
+  "main": "plugin.js",
+  "peerDependencies": {
+    "lighthouse": "^5.0.0"
+  }
+}
+```
+
+#### `plugin.js`
+
+This file contains the configuration for your plugin. It can be called anything you like, just ensure it is referenced by the `"main"` property in your `package.json`.
+
+**Example `plugin.js`**
+
+```js
+module.exports = {
+  // Additional audit to run on information Lighthouse gathered.
+  audits: [{path: 'lighthouse-plugin-cats/cat-audit.js'}],
```

Comment:
`lighthouse-plugin-cats/audits/has-cat-images.js`

mostly want to clarify the one-to-many category-to-audits relationship.